src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -12,11 +19,19 @@
 public class RobotMap {
 	public static class Port { // TODO: Correct Ports
 		public static class HumanInput {
-			public static final int joystick = 0;
-			public static final int xboxController = 1;
+			public static final int leftStick = -1;
+			public static final int xboxController = -1;
+			public static final int rightStick = -1;
```

Comment:
There is no reason to do this, because we set these values ourselves. Leave them as 0, 1, and 2.